tests/test_samples.py

```diff
@@ -415,6 +415,18 @@ def test_hist_levels():
                levels=[0.95, 0.68], bins=20)
 
 
+def test_plot_2d_no_axes():
+    np.random.seed(3)
+    ns = read_chains('./tests/example_data/pc')
+    ns[['x0', 'x1', 'x2']].plot_2d()
+
+
+def test_plot_1d_no_axes():
+    np.random.seed(3)
+    ns = read_chains('./tests/example_data/pc')
+    ns[['x0', 'x1', 'x2']].plot_1d()
```

Comment:
I've added these tests, but the `drop_labels()` ones are failing as expected. What would you expect for a mixture of labelled and unlabelled columns?

Reply:
If a parameter/column has a (tex) label, use that, if a parameter/column does not have a (tex) label then use the column handle.

Reply:
I've fixed this for the separate plots by only changing `name` to the label if the label exists, rather than setting it to a blank string as currently happens.

This solves the issues for individual plots (e.g. `plot.kde_1d()`), but not `plot_1d/2d()`, which use the label mapping to create the axes, which leads to blanks.